packages/workers-shared/router-worker/src/index.ts

```diff
@@ -98,7 +163,9 @@ export default {
 			}
 
 			// If we have a user-Worker, but no assets, dispatch to Worker script
-			const assetsExist = await env.ASSET_WORKER.unstable_canFetch(request);
+			const assetsExist = await env.ASSET_WORKER.unstable_canFetch(request, {
+				hasStaticRouting,
```

Comment:
hmm. the _intention_ of this API was that it would be exposed to users, but we never quite landed it (hence `unstable_`). I don't love having clients having to know anything about "hasStaticRouting". I wonder if we can obfuscate that at all? Either asset worker also gets this prop in its config, or we skirt something into the request? Idk.

Not blocking, because this isn't a public API. But if there is a tidier way to do it, I'd like to do that if we can.

Reply:
😬 How about stripping the `Sec-Fetch-Mode` header off?

Reply:
Yeah, of these options I think I like including `has_static_routing` in the asset configuration. Will update here